plugins/scaffolder/src/components/fields/RepoUrlPicker/AzureRepoPicker/README.md

```diff
@@ -0,0 +1,53 @@
+# Azure Repo Picker
+
+This field have `three` main behaviors:
+
+- (Default) Static without `allowedOrganizations` and `allowedOwners` it's will display only `InputTexts`.
+- (Default) Static with `allowedOrganizations` and `allowedOwners` it's will display `Selects` with predefined values.
+- Dynamic, that will fetch `organizations` and `owners` from `Azure DevOps Api` and display `Selects`.
+
+## Configuring this apiRef
+
+### On `package.json`
+
+```json
+{
+  "files": [
+    ...
+    "src/plugins/azure-devops-apiref/config.d.ts"
+  ],
+  "configSchema": "src/plugins/azure-devops-apiref/config.d.ts"
+}
+```
+
+### On `app-config.yaml`
+
+Configure api ref `defaultOrganization`
+
+```yaml
+app:
+  ...
+  azureDevOpsApiRef:
+    defaultOrganization: default-organization-of-token
+```
+
+and configure Standard `proxy to azure devops` route
+
+```yaml
+proxy:
+  ...
+  '/azure-devops-apiref':
+    target: "https://dev.azure.com"
+    headers:
+      Authorization: Basic ${PROXY_AZURE_DEVOPS_AUTH_HEADER_BASIC}
```

Comment:
The integrations section of the config already has an azure token set up. That's the one that the scaffolder uses to do actual work later on, or alternatively the user's own oauth token if that's passed along as a secret. We do not want the user to go through the trouble of setting up another token by hand, that might be configured differently or have other issues.

Reply:
hello, @freben , yes, this is the correct way, but try to think of it in a different way why rewrite a large amount of code in the backend with many layers between the frontend apiref and the backend api so that the azure library finally calls it Same API.

All these steps can be shortened, but if you, as an owner, apply this architecture, you will return to the table kkk.

Don't worry, thanks for your comments, I just wanted to give you another perspective on this code.

In a few days I will refactor my code.